configs/woocommerce-github-sync-labels/labels/commons.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "name": "changelog: add",
+    "color": "5c7588",
+    "description": "A new feature, function, or functionality was added."
+  },
+  {
+    "name": "changelog: breaking",
+    "color": "5c7588",
+    "description": "A change breaking previous API or functionality."
+  },
+  {
+    "name": "changelog: dev",
+    "color": "5c7588",
+    "description": "Developer-facing only change."
+  },
+  {
+    "name": "changelog: doc",
+    "color": "5c7588",
+    "description": "Updated customer- or developer-facing documentation"
+  },
+  {
+    "name": "changelog: fix",
+    "color": "5c7588",
+    "description": "Took care of something that wasn't working."
+  },
+  {
+    "name": "changelog: none",
+    "color": "5c7588",
+    "description": "Skip changelog entry for this PR"
+  },
+  {
+    "name": "changelog: tweak",
+    "color": "5c7588",
+    "description": "Small change, that isn't actually very important."
+  },
+  {
+    "name": "changelog: update",
+    "color": "5c7588",
+    "description": "Big changes to something that wasn't broken."
+  }
+]
```

Comment:
💅  Missing newline at the end of the file

<img width="481" alt="Screenshot 2022-08-02 at 13 47 37" src="https://user-images.githubusercontent.com/5908855/182345660-bc69ddf2-73f4-4250-8249-aa4f6d9b7fb5.png">